examples.py

```diff
@@ -11,6 +11,10 @@
 rsp = client.ticker('btcusd')
 print(rsp)
 
-print('[*] Calling hourli ticker endpoint for BTC/USD currency pair:')
-rsp = client.ticker('btcusd')
+print('[*] Calling hourly ticker endpoint for BTC/USD currency pair:')
+rsp = client.ticker('btceur')
+print(rsp)
+
+print('[*] Calling transactions in the last minute endpoint for BTC/USD currency pair:')
+rsp = client.transactions('btcusd', 'minute')
```

Comment:
please, lets call this as client.transactions('btcusd', period='minute') as its much more transparent for what it does.

Reply:
Resolved as i see no futher conflicts.